models/workitem_repository.go

```diff
@@ -132,6 +134,22 @@ func (r *GormWorkItemRepository) Create(ctx context.Context, typeID string, fiel
 		Type:   typeID,
 		Fields: Fields{},
 	}
+	publicKey, err := token.ParsePublicKey(token.RSAPublicKey)
```

Comment:
@aslakknutsen what is your objection here, concretely? (i.e why does it need to be moved?)